backend-shared/plannershared.cpp

```diff
@@ -100,12 +100,13 @@ void plannerShared::set_dive_mode(int value)
 
 int plannerShared::reserve_gas()
 {
-	return qPrefDivePlanner::reserve_gas();
+	return qPrefUnits::pressure() == units::BAR ? prefs.reserve_gas / 1000 :
+												  mbar_to_PSI(prefs.reserve_gas);
```

Comment:
again something odd going on with your indentation...

Reply:
I wonder if I have a wrong setting in qtCreator which I normally use for qml files

Reply:
we have sample settings for QtCreator in https://github.com/Subsurface-divelog/subsurface/blob/master/CODINGSTYLE.md#qtcreator 
Those may not be perfect, but they seem to work for me?
(BTW: this is a .cpp file, so maybe this was Xcode?)